charts/opentelemetry-demo/templates/NOTES.txt

```diff
@@ -9,33 +9,10 @@
  ╚═════╝    ╚═╝   ╚══════╝╚══════╝    ╚═════╝ ╚══════╝╚═╝     ╚═╝ ╚═════╝
 
 
-- Frontend UI: http://localhost:8080
+- All services are available via the Frontend proxy: http://localhost:8080
```

Comment:
Are there specific uris to access each service behind 8080?

Reply:
Added all exposed services to this note.